core/base/src/constants/finality.ts

```diff
@@ -76,6 +76,7 @@ const finalityThresholds = [
   ["BaseSepolia", 512],
   ["OptimismSepolia", 512],
   ["PolygonSepolia", 32],
+  ["MonadDevnet", 0],
```

Comment:
should this be a 1 like Fantom?

Reply:
I changed MonadDevent and Berachain to 1 and SnaxChain, Unichain and Worldchain to 512 per offline discussion with @evan-gray.

Reply:
concur, this is just to match the other OP Stack chains, not because it's factually accurate 👀